app/routes/app/brainstorm/index.tsx

```diff
@@ -153,6 +165,21 @@ type MarketplaceTableProps = {
 
 // Responsive layout for displaying marketplaces. On desktop, takes on a pseudo-table layout. On mobile, hide the header and become a list of self-contained cards.
 function MarketplacesTable({ marketplaces }: MarketplaceTableProps) {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const onSort = (header: string) => {
```

Comment:
I think this logic could be simplified. We don't care to check what the `sortBy` field is, only to toggle order if it's present.